src/main/java/EntryPoint/config/RedisConfig.java

```diff
@@ -0,0 +1,15 @@
+package EntryPoint.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.redis.connection.RedisConnection;
+import org.springframework.data.redis.connection.RedisConnectionFactory;
+import org.springframework.data.redis.core.StringRedisTemplate;
+
+@Configuration
+public class RedisConfig {
```

Comment:
Class name should be RedisTemplateConfig.

Reply:
resolved